src/psyclone/nemo.py

```diff
@@ -49,9 +49,8 @@
 from psyclone.core.access_type import AccessType
 from psyclone.psyGen import PSy, Invokes, Invoke, InvokeSchedule, Node, \
     Loop, CodedKern, InternalError, NameSpaceFactory, Schedule, \
-    Fparser2ASTProcessor
+    Fparser2ASTProcessor, colored, SCHEDULE_COLOUR_MAP
 from fparser.two.utils import walk_ast, get_child
```

Comment:
Not your fault but pylint points out that we already import `walk_ast` and `get_child` and that `copy` and `AccessType` are not used.

Reply:
Removed unnecessary imports